kolibri/core/webpack/templatetags/webpack_tags.py

```diff
@@ -18,7 +18,6 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 from django import template
-from django.utils.safestring import mark_safe
```

Comment:
Ohh.. I think I know what happened, I think I have to reinstall the precommit hooks after I had to wipe my kolibri dir.